ddl/column_test.go

```diff
@@ -288,22 +288,22 @@ func TestColumnBasic(t *testing.T) {
 	require.Equal(t, values[0].GetInt64(), int64(202))
 	require.Equal(t, values[4].GetInt64(), int64(101))
 
-	jobID = testDropColumnInternal(tk, t, testNewContext(store), tableID, "c1", false, dom)
+	jobID = testDropColumnInternal(tk, t, testkit.NewTestKit(t, store).Session(), tableID, "c1", false, dom)
 	testCheckJobDone(t, store, jobID, false)
 
-	jobID = testDropColumnInternal(tk, t, testNewContext(store), tableID, "c3", false, dom)
+	jobID = testDropColumnInternal(tk, t, testkit.NewTestKit(t, store).Session(), tableID, "c3", false, dom)
 	testCheckJobDone(t, store, jobID, false)
 
-	jobID = testDropColumnInternal(tk, t, testNewContext(store), tableID, "c4", false, dom)
+	jobID = testDropColumnInternal(tk, t, testkit.NewTestKit(t, store).Session(), tableID, "c4", false, dom)
 	testCheckJobDone(t, store, jobID, false)
 
-	jobID = testCreateIndex(tk, t, testNewContext(store), tableID, false, "c5_idx", "c5", dom)
+	jobID = testCreateIndex(tk, t, testkit.NewTestKit(t, store).Session(), tableID, false, "c5_idx", "c5", dom)
 	testCheckJobDone(t, store, jobID, true)
 
-	jobID = testDropColumnInternal(tk, t, testNewContext(store), tableID, "c5", false, dom)
+	jobID = testDropColumnInternal(tk, t, testkit.NewTestKit(t, store).Session(), tableID, "c5", false, dom)
 	testCheckJobDone(t, store, jobID, false)
 
-	jobID = testDropColumnInternal(tk, t, testNewContext(store), tableID, "c6", true, dom)
```

Comment:
Could we remove the function of `testNewContext`? And why?

Reply:
not now, there are still some other usage